content/docs/04-clusters/06-cluster-management/01.9-monitoring/20-deploy-agent.md

```diff
@@ -77,6 +77,16 @@ charts:
 8. Confirm your changes by selecting **Confirm & Create**.
 
 
+9. Click on **Add New Pack**.
+
+
+10. Use the following information to find the Prometheus Operator pack.
```

Comment:
10. "... to find the Spectro Cluster Metrics pack"

and 

6. "... to find the Prometheus Agent pack"